libopae/plugins/xfpga/sysfs.c

```diff
@@ -1883,8 +2022,27 @@ fpga_result make_sysfs_object(char *sysfspath, const char *name,
 	struct stat objstat;
 	int statres;
 	fpga_result res = FPGA_OK;
+	max_path_t object_paths[MAX_SYSOBJECT_GLOB];
```

Comment:
consumes quite a lot of stack space.  Is this necessary?

Reply:
let me think about it...